data/remnant ships.txt

```diff
@@ -372,6 +372,84 @@ ship "Gull" "Gull (Bunks)"
 
 
 
+ship "Ibis"
+	sprite "ship/ibis"
+	thumbnail "thumbnail/ibis"
+	attributes
+		category "Medium Warship"
+		licenses
+			"Remnant Capital"
+		"cost" 11627000
+		"shields" 14400
+		"hull" 6200
+		"required crew" 20
+		"bunks" 48
+		"mass" 350
+		"drag" 7.1
+		"heat dissipation" 0.7
+		"fuel capacity" 700
+		"ramscoop" 2
+		"cargo space" 103
+		"outfit space" 543
+		"weapon capacity" 171
+		"engine capacity" 162
+		"shield generation" 4.1
+		"shield energy" 3.6
+		"hull repair rate" 1.2
+		"hull energy" 0.95
+		"outfit scan power" 18
+		"outfit scan speed" 1
+		"tactical scan power" 39
+		weapon
+			"blast radius" 240
+			"shield damage" 2400
+			"hull damage" 1200
+			"hit force" 3000
+	outfits
+		"Inhibitor Cannon" 3
+		"Point Defense Turret" 2
+
+		"Aeon Cell"
+		"Crystal Capacitor" 3
+		"Emergency Ramscoop" 2
+		"Thermoelectric Cooler" 2
+		"Quantum Key Stone"
+		"Salvage Scanner"
+		"Tuning Rifle" 20
+
+		"Smelter-Class Thruster"
+		"Smelter-Class Steering"
+		"Forge-Class Steering"
+		"Scram Drive"
+		
+	drone 0 30 over 
+	drone -34 68 over
+	drone 0 81 over
+	drone 34 68 over
+	fighter -25 10 under
+	fighter 25 10 under
+	engine -16.5 120 0.7
+	engine 0 116.5 1
+	engine 16.5 120 0.7
+	gun 0 -135.5 "Inhibitor Cannon"
+	gun 15.5 -102 "Inhibitor Cannon"
+	gun -15.5 -102 "Inhibitor Cannon"
+	turret 0 -62 "Point Defense Turret"
+	turret 0 -20.5
+	turret 0 58.5 "Point Defense Turret"
+	leak "leak" 50 50
+	leak "flame" 50 80
+	leak "big leak" 90 30
+	explode "tiny explosion" 10
+	explode "small explosion" 25
+	explode "medium explosion" 25
+	explode "large explosion" 10
+	"final explode" "final explosion medium"
+	description "The Ibis is a multipurpose carrier that serves as a mobile mining platform, an escort ship for Pelican fleets, and a combat platform using Terns for screening. It is capable of fielding a single wing of support craft, consisting of two Petrels and four Terns."
+	description "	The Ibis was designed shortly after the Albatross as a means of extending the range of resource collection efforts after the Remnant realized that an Alpha invasion might not be as imminent as originally believed."
```

Comment:
`as a means`, is this supposed to be plural?

Reply:
Yes, this is what is called a stock expression, in 'as a means'  the word means is always plural.
(Source: am an English major)

Reply:
yep, resolved